workspace/__ardulib__/ST7735/XST7735.h

```diff
@@ -7,8 +7,16 @@
 
 #define DELAY 0x80
 
-#define ST7735_TFTWIDTH 128
-#define ST7735_TFTHEIGHT 160
+#define ST7735_TFTWIDTH_128 128
+#define ST7735_TFTHEIGHT_160 160
```

Comment:
Why adding a value to the constant name?

Reply:
For the future upgrades. For displays with the same driver but another resolution.